lib/elasticquery/base.rb

```diff
@@ -1,11 +1,11 @@
 require 'active_support/core_ext/class/attribute'
 require 'elasticquery/query'
-require 'elasticquery/queries/all'
+require 'elasticquery/builder'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.